soh/soh/Enhancements/randomizer/static_data.cpp

```diff
@@ -199,6 +200,7 @@ std::unordered_map<RandomizerHint, StaticHintInfo> StaticData::staticHintInfoMap
   {RH_DAMPES_DIARY,        StaticHintInfo(HINT_TYPE_AREA,     {RHT_DAMPE_DIARY},                   RSK_DAMPES_DIARY_HINT,   true, {}, {RG_PROGRESSIVE_HOOKSHOT},    {RC_DAMPE_HINT})},
   {RH_GREG_RUPEE,          StaticHintInfo(HINT_TYPE_AREA,     {RHT_GREG_HINT},                     RSK_GREG_HINT,           true, {}, {RG_GREG_RUPEE},              {RC_GREG_HINT})},
   {RH_SARIA_HINT,          StaticHintInfo(HINT_TYPE_AREA,     {RHT_SARIA_TALK_HINT, RHT_SARIA_SONG_HINT}, RSK_SARIA_HINT,   true, {}, {RG_PROGRESSIVE_MAGIC_METER}, {RC_SARIA_SONG_HINT, RC_SONG_FROM_SARIA}, true)},
+  {RH_LOACH_HINT,          StaticHintInfo(HINT_TYPE_AREA,     {RHT_LOACH_HINT},                    RSK_SARIA_HINT,          true, {RC_LH_HYRULE_LOACH})},
```

Comment:
This should refer to it's own hint setting, not sarias, and should be HINT_TYPE_ITEM, not HINT_TYPE_AREA, as it is giving an item name not a area.